From 0b52e7d664def49e868d576c81c4e8b4870c1975 Mon Sep 17 00:00:00 2001
From: Cheng Wang2 <cheng.wang2@amlogic.com>
Date: Thu, 12 Sep 2024 14:05:03 +0800
Subject: [PATCH] T7C: recovery show on panel [1/1]

PD#SWPL-185484

Problem:
The recovery screen does not display if
there is no HDMI TX connection.

Solution:
config to sec connector

Verify:
t7c

Change-Id: I7b777a1d6b2c88c7942a0fa601965e60b86f01ee
Signed-off-by: Cheng Wang2 <cheng.wang2@amlogic.com>
---
 minui/graphics_drm.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/minui/graphics_drm.cpp b/minui/graphics_drm.cpp
index 1ad51801..b9c146ca 100644
--- a/minui/graphics_drm.cpp
+++ b/minui/graphics_drm.cpp
@@ -391,14 +391,14 @@ bool MinuiBackendDrm::FindAndSetMonitor(int fd, drmModeRes* resources) {
   std::vector<drmModeConnector*> drmConnectors;
 
   /* Amlogic */
-  const char* uboot_first_mode = get_bootloader_env("outputmode");
+  const char* uboot_first_mode = get_bootloader_env("outputmode2");
   if (uboot_first_mode) {
 	printf("Recovery uses uboot mode: %s\n", uboot_first_mode);
     auto envConnectors = find_used_connector_by_uboot_mode(fd, resources, uboot_first_mode);
 
     // for multi-screen
     if (envConnectors.size() >= 2) {
-        const char* connector0_type = get_bootloader_env("connector0_type");
+        const char* connector0_type = get_bootloader_env("connector1_type");
         drm_connector_type_t extend_connector_type = drmStringToConnType(connector0_type);
         printf("find first connector %s  %d\n", connector0_type, extend_connector_type);
         for (auto connector : envConnectors) {
-- 
2.42.0

